src/Kodify/BlogBundle/Resources/views/Post/List/index.html.twig

```diff
@@ -3,10 +3,10 @@
 {% block content %}
 
     {% for post in posts %}
-        <div class="panel panel-default">
+        <div class="panel panel-default col-md-6">
```

Comment:
Nice solution :)

If you accessed this page from a mobile device, how do you think you'll see the posts?